From 7e36d5c221b732c6d536581843c3d7f976bb2331 Mon Sep 17 00:00:00 2001
From: "cary.sc.chen" <cary.sc.chen@deltaww.com>
Date: Fri, 19 Nov 2021 09:53:14 +0800
Subject: [PATCH] Modify tn48m/tn48m-dn series dts for Linux-5.10.7x

update tn48m/tn48m-dn series device tree with "dma-ranges" attributes to work with Linux 5.10.7x

Signed-off-by: cary.sc.chen <cary.sc.chen@deltaww.com>
---
 arch/arm64/boot/dts/marvell/delta-tn4810m-dn.dts | 1 +
 arch/arm64/boot/dts/marvell/delta-tn4810m.dts    | 1 +
 arch/arm64/boot/dts/marvell/delta-tn48m-dn.dts   | 1 +
 arch/arm64/boot/dts/marvell/delta-tn48m.dts      | 1 +
 4 files changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/delta-tn4810m-dn.dts b/arch/arm64/boot/dts/marvell/delta-tn4810m-dn.dts
index f5fd373dc..4ad9bf4cf 100644
--- a/arch/arm64/boot/dts/marvell/delta-tn4810m-dn.dts
+++ b/arch/arm64/boot/dts/marvell/delta-tn4810m-dn.dts
@@ -1696,6 +1696,7 @@
 };
 
 &cp0_pcie0 {
+	dma-ranges = <0x42000000 0x0 0x00000000 0x0 0x00000000 0x0 0x40000000>;
 	ranges = <0x81000000 0x0 0xfb000000 0x0 0xfb000000 0x0 0xf0000
 		0x82000000 0x0 0xf6000000 0x0 0xf6000000 0x0 0x2000000
 		0x82000000 0x0 0xf9000000 0x0 0xf9000000 0x0 0x100000>;
diff --git a/arch/arm64/boot/dts/marvell/delta-tn4810m.dts b/arch/arm64/boot/dts/marvell/delta-tn4810m.dts
index c3de8f61b..ac509ee52 100644
--- a/arch/arm64/boot/dts/marvell/delta-tn4810m.dts
+++ b/arch/arm64/boot/dts/marvell/delta-tn4810m.dts
@@ -1696,6 +1696,7 @@
 };
 
 &cp0_pcie0 {
+	dma-ranges = <0x42000000 0x0 0x00000000 0x0 0x00000000 0x0 0x40000000>;
 	ranges = <0x81000000 0x0 0xfb000000 0x0 0xfb000000 0x0 0xf0000
 		0x82000000 0x0 0xf6000000 0x0 0xf6000000 0x0 0x2000000
 		0x82000000 0x0 0xf9000000 0x0 0xf9000000 0x0 0x100000>;
diff --git a/arch/arm64/boot/dts/marvell/delta-tn48m-dn.dts b/arch/arm64/boot/dts/marvell/delta-tn48m-dn.dts
index 59b5d3d7f..522de235f 100644
--- a/arch/arm64/boot/dts/marvell/delta-tn48m-dn.dts
+++ b/arch/arm64/boot/dts/marvell/delta-tn48m-dn.dts
@@ -358,6 +358,7 @@
 };
 
 &cp0_pcie0 {
+	dma-ranges = <0x42000000 0x0 0x00000000 0x0 0x00000000 0x0 0x40000000>;
 	ranges = <0x81000000 0x0 0xfb000000 0x0 0xfb000000 0x0 0xf0000
 		0x82000000 0x0 0xf6000000 0x0 0xf6000000 0x0 0x2000000
 		0x82000000 0x0 0xf9000000 0x0 0xf9000000 0x0 0x100000>;
diff --git a/arch/arm64/boot/dts/marvell/delta-tn48m.dts b/arch/arm64/boot/dts/marvell/delta-tn48m.dts
index 70ff16000..90b2b5a63 100644
--- a/arch/arm64/boot/dts/marvell/delta-tn48m.dts
+++ b/arch/arm64/boot/dts/marvell/delta-tn48m.dts
@@ -358,6 +358,7 @@
 };
 
 &cp0_pcie0 {
+	dma-ranges = <0x42000000 0x0 0x00000000 0x0 0x00000000 0x0 0x40000000>;
 	ranges = <0x81000000 0x0 0xfb000000 0x0 0xfb000000 0x0 0xf0000
 		0x82000000 0x0 0xf6000000 0x0 0xf6000000 0x0 0x2000000
 		0x82000000 0x0 0xf9000000 0x0 0xf9000000 0x0 0x100000>;
-- 
2.17.1

